browser/go/conversion-test/main.go

```diff
@@ -0,0 +1,673 @@
+// +build js, wasm
+
+package main
+
+import (
+	"math/big"
+	"syscall/js"
+	"time"
+
+	"github.com/0xProject/0x-mesh/browser/go/jsutil"
+	"github.com/0xProject/0x-mesh/common/types"
+	"github.com/0xProject/0x-mesh/constants"
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/0xProject/0x-mesh/zeroex/ordervalidator"
+	"github.com/0xProject/0x-mesh/zeroex/orderwatch/decoder"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	loadEventName = "0xmeshtest"
+)
+
+func main() {
+	setGlobals()
+	triggerLoadEvent()
+	select {}
+}
+
+func setGlobals() {
+	conversionTestCases := map[string]interface{}{
+		"contractEventsAsync": js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+			return jsutil.WrapInPromise(func() (interface{}, error) {
```

Comment:
I don't think we actually need to use `jsutil.WrapInPromise` here. We're just returning some values and it doesn't look like we're doing anything blocking/asynchronous like sending network requests or reading a file. If we change these functions so that they don't return a promise, we should also rename them by removing the `Async` suffix.